test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,18 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        assertIsAnyNull("abc", null);
+        assertIsAnyNull(1, "abc", new LinkedList<Integer>(), null);
+
+        assertNoneIsNull("abc", new Scanner(System.in));
+        assertNoneIsNull(44, false, new HashMap<String, Integer>());
+
+    }
+
+    private void assertIsAnyNull(Object... objects) { assertTrue(Utils.isAnyNull(objects));}
```

Comment:
For specific test methods, you may try to name as per the 3-part naming process mentioned in the tutorial.